io.catenax.iot_sensor_data/2.0.0/IotSensorData.ttl

```diff
@@ -0,0 +1,174 @@
+   samm:description "Characteristic to describe the property battery level."@en ;
+   samm:dataType xsd:decimal ;
+   samm-c:unit unit:percent .
+
+:SensorDataCharacteristics a samm-c:List ;
+   samm:preferredName "Sensor Data Characteristics"@en ;
+   samm:description "Characteristic to describe the related Sensor data."@en ;
+   samm:dataType :SensorDataEntity .
+
+:TransmissionMethodCharacteristic a samm:Characteristic ;
+   samm:preferredName "Transmission Method Characteristic"@en ;
+   samm:description "Characteristic that describes the property transmission method."@en ;
+   samm:dataType xsd:string .
+
+:SensorGeoLocationEntity a samm:Entity ;
+   samm:preferredName "Sensor Geo Location Entity"@en ;
+   samm:description "Encapsulates the Geo Data relevant attributes."@en ;
+   samm:properties ( :latitude :longitude :altitude :geoDataTimestamp ) .
+
+:SensorDataEntity a samm:Entity ;
+   samm:preferredName "Sensor Data Entity"@en ;
+   samm:description "Encapsulates the Sensor Data relevant attributes."@en ;
+   samm:properties ( :sensorType :sensorValue :sensorUnit ) .
+
+:latitude a samm:Property ;
+   samm:preferredName "Latitude"@en ;
+   samm:description "The angle between zenith and a plane parallel to the equator."@en ;
+   samm:characteristic :LatitudeCharacteristic ;
+   samm:exampleValue "40.20361"^^xsd:decimal .
+
+:longitude a samm:Property ;
+   samm:preferredName "Longitude"@en ;
+   samm:description "Geographic coordinate that specifies the east-west position of a point on the Earth's surface."@en ;
+   samm:characteristic :LongitudeCharacteristic ;
+   samm:exampleValue "11.3102"^^xsd:decimal .
+
+:altitude a samm:Property ;
+   samm:preferredName "Altitude"@en ;
+   samm:description "Antenna Altitude above/below mean-sea-level (geoid)."@en ;
+   samm:characteristic :AltitudeCharacteristic ;
+   samm:exampleValue "280.20"^^xsd:decimal .
+
+:geoDataTimestamp a samm:Property ;
+   samm:preferredName "Geo Data Timestamp"@en ;
+   samm:description "The timestamp of the latest sensor reading of the geo data."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-02-04T14:48:54"^^xsd:dateTime .
+
+:sensorType a samm:Property ;
+   samm:preferredName "Sensor Type"@en ;
+   samm:description "Different types of sensors that are commonly used in various applications, measuring one of the physical properties like Temperature, Pressure,  Resistance, Shock, Conduction, Heat Transfer etc."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Temperature" .
+
+:sensorValue a samm:Property ;
+   samm:preferredName "Sensor Value"@en ;
+   samm:description "The measured value of the sensor type."@en ;
+   samm:characteristic :SensorValueCharacteristics ;
+   samm:exampleValue "32.00"^^xsd:decimal .
+
+:sensorUnit a samm:Property ;
+   samm:preferredName "Sensor Unit"@en ;
+   samm:characteristic samm-c:UnitReference ;
+   samm:exampleValue "Celsius"^^samm:curie .

```

Comment:
The correct example value should be `unit: Celsius`

Reply:
done.

Reply:
Sorry, my mistake. value should be without any space. `unit:Celsius`